app/src/main/res/layout/activity_chat.xml

```diff
@@ -61,14 +61,19 @@
             android:hint="Message"
             android:inputType="text"
             />
+
         <ImageButton
             android:id="@+id/send_message_button"
+            android:onClick="sentMessage"
```

Comment:
can we name it like button_send_message? that's how we name it in other xml files.